examples/encoding/multiple_audio_dash_mpd_auto_analysis.py

```diff
@@ -0,0 +1,413 @@
+                                                  segment_path='video/360p/')
+    fmp4_representation_360p = bitmovin.manifests.DASH.add_fmp4_representation(object_=fmp4_representation_360p,
+                                                                               manifest_id=dash_manifest.id,
+                                                                               period_id=period.id,
+                                                                               adaptationset_id=video_adaptation_set.id
+                                                                               ).resource
+
+    fmp4_representation_240p = FMP4Representation(FMP4RepresentationType.TEMPLATE,
+                                                  encoding_id=encoding.id,
+                                                  muxing_id=video_muxing_240p.id,
+                                                  segment_path='video/240p/')
+    fmp4_representation_240p = bitmovin.manifests.DASH.add_fmp4_representation(object_=fmp4_representation_240p,
+                                                                               manifest_id=dash_manifest.id,
+                                                                               period_id=period.id,
+                                                                               adaptationset_id=video_adaptation_set.id
+                                                                               ).resource
+
+    for information in audio_muxing_mpd_information:
+        lang = information.get('lang')
+        channel_layout = information.get('channel_layout')
+        muxing_id = information.get('muxing_id')
+        segment_path = information.get('segment_path')
+
+        audio_adaptation_set_lang_name = '{} {}'.format(lang, channel_layout).upper()
+
+        audio_adaptation_set = AudioAdaptationSet(lang=audio_adaptation_set_lang_name)
+        audio_adaptation_set = bitmovin.manifests.DASH.add_audio_adaptation_set(
+            object_=audio_adaptation_set,
+            manifest_id=dash_manifest.id,
+            period_id=period.id).resource
+
+        fmp4_representation_audio = FMP4Representation(FMP4RepresentationType.TEMPLATE,
+                                                       encoding_id=encoding.id,
+                                                       muxing_id=muxing_id,
+                                                       segment_path=segment_path)
+        fmp4_representation_audio = bitmovin.manifests.DASH.add_fmp4_representation(
+            object_=fmp4_representation_audio,
+            manifest_id=dash_manifest.id,
+            period_id=period.id,
+            adaptationset_id=audio_adaptation_set.id
+        ).resource
+
+    print('Starting DASH Manifest creation...')
+    bitmovin.manifests.DASH.start(manifest_id=dash_manifest.id)
+
+    try:
+        bitmovin.manifests.DASH.wait_until_finished(manifest_id=dash_manifest.id)
```

Comment:
Maybe a little bit of imformation about the generated MPD would be great. (E.g. print out mpd url)
So you can test your encoding when everything is finished. But beside this the example looks really nice.